README.md

```diff
@@ -1,36 +1,60 @@
 
-## Getting Started
-To contribute to the `pds-ember` package (currently the only package in this mono repo) please see the [Contributing](packages/pds-ember/CONTRIBUTING.md) guide for details.
+But don't worry, if you need to use the old project, there are a couple of "frozen in time" branches that you can use (and potentially even release from, if you need). See *"Which version to use?"* below.
+
+### Which version to use?
+
+The current version of this package is the one used for the "managed sunsetting" of the library from our products. As such, in time some components are deprecated and then removed from the repository, and some surgical patches to the code are done when necessary to facilitate the migration to the new UI library ([HashiCorp Design System components](https://github.com/hashicorp/design-system)).
+
+**If for any reasons you need to stay on an older version**, you can:
+- use the package version `0.6.4` to stay "frozen in time" at the moment in which the "maintained sunsetting" phase has started.
+- use the package version `1.x.x` to stay on a release path that sees only minor patches but not components removal
+  - in this case, if you need to make changes, use the branch `master-version-1.x` and release to NPM from that branch
+
+---
 
+*Old introduction to the project, left for reference:*
 
-## Creating + Consuming a Release
+> Pds stands for Product Design System.  The original intent was to rename `structure` to Pds because the developer at the time thought it was more clear.  Unfortunately, that renaming never happened so there are two names.
+> This is a mono repo with the idea that the Ember, React, etc projects scattered around Hashi could all eventually rely on a single source for design components.  Currently there is only `pds-ember`.
 
-### Update Version
-1. `yarn workspace @hashicorp/pds-ember version patch` to bump the version by `0.0.1`
+## Getting started
+To contribute to the `pds-ember` package (currently the only package in this mono repo) please see the [Contributing](packages/pds-ember/CONTRIBUTING.md) guide for details.
+
+## Creating + Consuming a release
+
+### Update version
+1. `yarn workspace @hashicorp/pds-ember version patch` to bump the version by `2.0.2`
 1. Git add the yml file and the version update generated by the command.
 1. Create a new PR for this and get it approved and merged into main.
 1. Checkout main branch, pull down the new changes.
 
-### Update Git Version
-1. Use `git tag [PACKAGE-NAME]-[VERSION]` e.g. `git tag pds-ember-0.5.2`
+### Update Git version
+1. Use `git tag [PACKAGE-NAME]-[VERSION]` e.g. `git tag pds-ember-2.0.2`
 1. `git push --tags`
 
 ### Update NPM
 1. Make sure your NPM account has been added to the Hashi group (check with IT)
 1. `yarn workspace @hashicorp/[PACKAGE-NAME] npm publish` for example `yarn workspace @hashicorp/pds-ember npm publish`
```

Comment:
@Dhaulagiri which command did you use to publish yesterday?
- `yarn workspace @hashicorp/pds-ember npm publish` in the root folder
- or just `npm publish` in the `packages/pds-ember` folder?

(asking because yesterday the first was not working for me, while the second worked at the first attempt this morning)

Reply:
So weird, now also the first command worked flawlessly 🤷‍♂️

Reply:
I used the former but the two commands you listed should be equivalent